### ISSUE TITLE
Accessibility: newsubprojects.html

### ISSUE THREAD

**Issue by falyne** [jupyterhub/outreachy#70]
### What page is this for?

https://jupyter.org/governance/newsubprojects.html

### WAVE accessibility report

![newssubprojects](https://user-images.githubusercontent.com/19360782/196050153-6ab2c6ec-73e7-4df3-b8a3-ddd1fe40e0a4.PNG)

Evaluation
**9 ERRORS**
-**5 Empty form label errors** because the search input and the search button have a label associated with them but contain no content.
To improve accessibility we need to Ensure that the form label contains text that describes the function of the associated form control. A descriptive title attribute may be added to the form control.
-**Two errors of Multiple form labels** . This indicate that multiple form labels are tied to the same input. This can affect screen reader users and mouse readers (when clicked on a search form the browser can get confused)
To improve accessibility to this forms we need to give unique ids to the form inputs
-**One empty Button.** This indicate that A button is empty or has no value text.
To Improve on accessibility, we need to Place text content within the element or give the element a value attribute.
-**Language missing or invalid**-The language of the document is not identified or a lang attribute value is invalid
the solution to this is to dentify the document language using the <html lang> attribute with a valid value (e.g., <html lang="en">). Ensure that all lang attribute values are valid.

**CONTRAST ERRORS**
-**Very Low contrast error found on the page** -Adequate contrast of text is necessary for all users, especially users with low vision
To improve on this issue, there need to be an Increase in the contrast between the foreground (text) color and the background color. Large text does not require as much contrast as smaller text.

**6 ALERTS**
**1 Suspicious link text- Some Link text contains extraneous text or may not make sense out of context.**
The solution is to re-write link text to be more descriptive and to ensure the absence of any extraneous text
Links, which are often read out of context, should clearly describe the destination or function of the link.
Ambiguous text, text that does not make sense out of context, and extraneous text can cause confusion and should be avoided.
**5 Redundant link Adjacent Links go to the same URL**
The solution is to combine the redundant links into one link and remove any redundant text or alternative text (for example, if a product image and product name are in the same link, the image can usually be given alt="").
**FEATURES**
**1-Alternative text is present for an image that is within a link.**
This presents the content of the image and/or the function of the link.
**-2 Form Label** -For Form Labels on the page the label is accurate, descriptive, succinct, and it is associated with the correct form control element.
**STRUCTURAL ELEMENTS**
-The structure of the Elements of the page shows a right placement/flow of element Hierarchically. Each elements rightly placed
-The Page has most accessibility features for a vast group of people
-ARIA labels which is present define accessible names to be read by screen readers for interface elements.
-ARIA role, state, or property present on the page provides enhanced semantics and accessibility for web content

### The HTML element to be changed

```html
<!--Language missing or invalid-->
<html>
<!--5 Empty form label errors-->
<label for="toctree-checkbox-1">
<i class="fas fa-chevron-down"></i>
</label>

<label for="toctree-checkbox-2">
<i class="fas fa-chevron-down"></i>
</label>

<label for="toctree-checkbox-3">
<i class="fas fa-chevron-down"></i>
</label>

<label for="__navigation" class="headerbtn" data-toggle="tooltip" data-placement="right" title="" data-original-title="Toggle navigation">
<span class="headerbtn__icon-container">
<i class="fas fa-bars"></i>
</span>
</label>

<label for="__page-toc" class="headerbtn headerbtn-page-toc">
<span class="headerbtn__icon-container">
<i class="fas fa-list"></i>
</span>
</label>
<!--Multiple Form 'labels-->
<input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation" aria-label="Toggle navigation sidebar">
<input type="checkbox" class="sidebar-toggle" name="__page-toc" id="__page-toc" aria-label="Toggle in-page Table of Contents">


<!--One empty Button-->
<button onclick="toggleFullScreen()" class="headerbtn" data-toggle="tooltip" data-placement="bottom" title="" data-original-title="Fullscreen mode">
<span class="headerbtn__icon-container">
<i class="fas fa-expand"></i>
</span>
</button>

<!--Very Low contrast error found on the page-->

code {
       font size: 87.5%;
       color : #83e8c;
       word-wrap : break-word;
     }

Be developed according to the Jupyter governance and contribution model that is documented
<a class="reference external" href="https://github.com/jupyter/governance">
here
</a>

<!--Redundant link-->
Copyright notices in individual files should be updated to the standard form described in our
<a class="reference external" href="https://raw.githubusercontent.com/jupyter/governance/master/projectlicense.md">
Project Jupyter License
</a>

If a Subproject team knows from the start that it would eventually like its Subproject to be part of one of the official Jupyter organizations, it can be incubated in the
<a class="reference external" href="https://github.com/jupyter-incubator">
jupyter-incubator
</a>

The goals of the incubation in the
<a class="reference external" href="https://github.com/jupyter-incubator">
jupyter-incubator
</a>

A Subproject team can initiate the
Redundant link
<a class="reference external" href="https://github.com/jupyter-incubator">
jupyter-incubator
</a>

incubation process by submitting a proposal for incubation. This process is designed to be lightweight and quick:
</p>
```

### Your proposed HTML (or CSS), after change

```html
<html lang="en">

<label for="toctree-checkbox-1" aria-label="checkbox1">
<i class="fas fa-chevron-down"></i>
</label>

<label for="toctree-checkbox-2" aria-label="checkbox2">
<i class="fas fa-chevron-down"></i>
</label>

<label for="toctree-checkbox-3" aria-label="checkbox3">
<i class="fas fa-chevron-down"></i>
</label>

<label for="__navigation" class="headerbtn" data-toggle="tooltip" data-placement="right" title="" data-original-title="Toggle navigation" aria-label="navigation">
<span class="headerbtn__icon-container">
<i class="fas fa-bars"></i>
</span>
</label>

<label for="__page-toc" class="headerbtn headerbtn-page-toc" aria-label="page-toc">
<span class="headerbtn__icon-container">
<i class="fas fa-list"></i>
</span>
</label>

<input type="checkbox" class="sidebar-toggle" name="__navigation" id="__navigation" aria-label="Toggle navigation sidebar" aria-labelledby="_navigation">
<input type="checkbox" class="sidebar-toggle" name="__page-toc" id="__page-toc" aria-label="Toggle in-page Table of Contents" aria-labelledby="_page-toc">

<button onclick="toggleFullScreen()" class="headerbtn" data-toggle="tooltip" data-placement="bottom" title="" data-original-title="Fullscreen mode" aria-label="Toggle Full Screen>
<span class="headerbtn__icon-container">
<i class="fas fa-expand"></i>
</span>
</button>

<!--in pydata-sphinx-theme.css?digest=1999514e3f237ded88cf-->
code {
       font size: 87.5%;
       color : #9e2a5f;
       word-wrap : break-word;
     }


Be developed according to the Jupyter governance and contribution model that is documented
<a class="reference external" href="https://github.com/jupyter/governance">
in the governance page
</a>

Copyright notices in individual files should be updated to the standard form described in our
<a class="reference external" href="https://raw.githubusercontent.com/jupyter/governance/master/projectlicense.md">
Jupyter Project License
</a>

If a Subproject team knows from the start that it would eventually like its Subproject to be part of one of the official Jupyter organizations, it can be incubated in the
<a class="reference external" href="https://github.com/jupyter-incubator">
jupyter-incubator
</a>

The goals of the incubation in the
Redundant link
<a class="reference external" href="https://github.com/jupyter-incubator">
Incubator
</a>

A Subproject team can initiate the
<a class="reference external" href="https://github.com/jupyter-incubator">
jupyter-incubator process
</a>
by submitting a proposal for incubation. This process is designed to be lightweight and quick:
</p>
```

**Comment by minrk** [comment]
Thanks for the suggestions!

**Comment by falyne** [comment]
Thanks @minrk

**Comment by sgibson91** [comment]
Thank you for your contribution! Since the contribution period is now over, we will close this issue.
